seatunnel-engine/seatunnel-engine-server/src/main/java/org/apache/seatunnel/engine/server/rest/RestHttpGetCommandProcessor.java

```diff
@@ -300,6 +296,54 @@ private void handleJobInfoById(HttpGetCommand command, String uri) {
         }
     }
 
+    private void handleJobInfoByJobId(HttpGetCommand command, String uri) {
```

Comment:
I think we can reuse `handleJobInfoById` method, not create new one.

Reply:
OK